### ISSUE TITLE
Suggestion: Add Components Folder

### ISSUE THREAD

**Issue by ayezioro** [mostaphaRoudsari/ladybug#409]
Hi,
There are already a few components (image viewer, item selector, FalseStartToggle) that need to be located at the components folder. Users usually, without knowing put all components together in the Users folder.
My suggestion is to add this folder in the github and maybe in the installation.
In addition or alternatively, in the instructions file to specify the files that need to be located in the components folder.
Thanks,
-A.

**Comment by MahmoudAbdelRahman** [comment]
Hi.
I usually drop them all into grasshopper canvas and each conponent goes automatically to its  location. Is this a good practice ?

**Comment by ayezioro** [comment]
@MahmoudAbdelRahman,
Yes. Probably this is one way ... that for my own personal preferences i don't like/use.
For ghuser files if i drop on canvas, and i don't uninstall before, i get multiple copies on the tabs.
Gha files are fine using the drop way.
Usually i select from the zip file the contents and drop on the user folder.
-A.

**Comment by chriswmackey** [comment]
@ayezioro and @MahmoudAbdelRahman ,
I'd like to hear @mostaphaRoudsari 's thoughts on this before implementing anything as he set up the original structure of putting everything in the userObjects folder.
This also means that we will have to revise all of our install instructions that currently say that you should just drag/drop all components onto the canvas but this can be done if it would make things easier from your perspective.

**Comment by mostaphaRoudsari** [comment]
Thank you @ayezioro for the suggestion. It's a good idea and makes everything cleaner.
This is the list of tasks that we need to address if we are going to make this change:
- [ ] installation process
- [ ] uninstall script
- [ ] update ladybug component
- [ ] update honeybee component - I have been thinking for some time to merge these two components into 1.
